### ISSUE TITLE
Detected package downgrade: Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson from 5.0.1 to 3.0.0 after updating to net6.0

### ISSUE THREAD

**Issue by ilya-git** [Azure/azure-functions-signalrservice-extension#285]
I have upgraded my Azure Function code to net6.0, and I started getting this warning:

>   SignalR.Relay.csproj: [NU1605] Detected package downgrade: Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson from 5.0.1 to 3.0.0. Reference the package directly from the project to select a different version. 
 SignalR.Relay -> Microsoft.Azure.WebJobs.Extensions.SignalRService 1.6.0 -> Microsoft.Azure.SignalR.Management 1.11.0 -> Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson (>= 5.0.1) 
 SignalR.Relay -> Microsoft.Azure.WebJobs.Extensions.SignalRService 1.6.0 -> Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson (>= 3.0.0)

Will there be some issues because of this and I have to directly reference the package or it should be fine? I suppose though the warning should not happen at all?

**Comment by Y-Sindo** [comment]
Duplicate #280.   This is a bug fixed in 1.7.0-beta.1. For 1.6.0, you could add 
```
<PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="5.0.1" />
```
to mitigate the issue.

**Comment by ilya-git** [comment]
Thanks, I'll give the beta a try. Do you have some timeline for it's release?

**Comment by Y-Sindo** [comment]
Fixed in 1.7.0
